src/runtime/HalideBuffer.h

```diff
@@ -885,6 +932,7 @@ class Buffer {
      * take ownership of the data, and does not set the host_dirty flag. */
     template<typename Array, size_t N>
     explicit Buffer(Array (&vals)[N]) {
+        // TODO: this could probably be made constexpr
```

Comment:
IMO either handle the TODO now or remove it. If the compiler can constant-fold it, it will. There shouldn't be any benefit in explicitly marking it as constexpr.

Reply:
Done (comment removed)